docs/source/api.rst

```diff
@@ -403,6 +403,12 @@ Low-Level Functionality
 
         The global py-evm chain instance.
 
+    .. method:: enable_fast_mode(flag: bool = True) -> None:
+
+        Enable or disable fast mode. This can be useful for speeding up tests.
```

Comment:
maybe state here that fast mode is experimental and may break other features of boa (like coverage)

Reply:
I've added a whole section dedicated to fast mode, lmk if you think I should add a warning here as well or if it would be redundant.